apps/src/lab2/views/components/versionHistory/version-history.module.scss

```diff
@@ -21,7 +21,8 @@
   justify-content: space-between;
 }
 
-.versionHistoryTitle {
+// This is duplicated to ensure it takes specificity over the default style.
+.versionHistoryTitle.versionHistoryTitle {
```

Comment:
@molly-moen - I noticed on `prod` in `pythonlab` projects that the color of the 'Version History' title is not white.
![Screenshot 2025-04-02 at 9 14 39 AM](https://github.com/user-attachments/assets/cf205528-70ca-49de-80a4-d4e2f71af873)

So updated style here. After update:

![Screenshot 2025-04-02 at 9 41 18 AM](https://github.com/user-attachments/assets/ec586adc-9f56-4ef8-b491-25864bc12d49)

Reply:
Yep I noticed this was happening too, see [this thread](https://codedotorg.slack.com/archives/C0532LCK125/p1743547172298639). I have a pr in progress to convert this to use the dark theme, so I'll handle it there!